src/Stormpath/Http/HttpClientRequestExecutor.php

```diff
@@ -91,4 +91,9 @@ private function addQueryString(array $queryString, RequestInterface $request)
         }
     }
 
+    public function getApiKey()
```

Comment:
if we move it to the dataStore, we will still need a public method to access it as the resources use getDataStore() to have access to the datastore.  because no resource extends dataStore, we will have to leave it public